grails-app/views/projectActivity/_sites.gsp

```diff
@@ -38,12 +38,12 @@
                     <div class="span2 text-right">
                         <span data-bind="if: added()">
                             <small>
-                                <a href="#" data-bind="click: removeSite" class="btn btn-small btn-danger">Remove</a>
+                                <a href="#" data-bind="click: removeSite" class="btn btn-small btn-danger" alt="Remove">&lt;&lt;</a>
```

Comment:
The attribute should be 'title', not 'alt' for an anchor. Alt is not a supported attribute for <a>.